bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/StaticImageImpl.java

```diff
@@ -99,4 +107,26 @@ public String getAltText() {
     public String getDataRef() {
         return null;
     }
+
+    @Override
+    @NotNull
+    protected FormComponentData getComponentData() {
```

Comment:
why this needs to be override?

Reply:
To add one extra property in case of image. i.e. LinkURL
I've added all the property for each component in the JIRA description.
https://jira.corp.adobe.com/browse/FORMS-5839

Reply:
removed